docs_source/source/documentation.rst

```diff
@@ -87,142 +91,152 @@ To install AxonDeepSeg, you just need to install it with ``pip`` using the follo
-The segmented acquisition itself will be saved in the same folder as the acquisition image,
-with the suffix 'segmented_', in png format.
+-o OVERLAP          Overlap value (in pixels) of the patches when doing the segmentation. 
+                    Higher values of overlap can improve the segmentation at patch borders, but also increase the segmentation time. Default value: 25. Recommended range of values: [10-100]. 
 
+.. NOTE ::
+   You can get the detailed description of all the arguments of the **axondeepseg** command at any time by using the **-h** argument:
+   ::
+    axondeepseg -h
 
-* To segment multiple images acquired with the same resolution, put them all in the same folder and
-launch the segmentation of this folder, like below::
+Segment a single image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    axondeepseg -t SEM -i test_sem_image/image 1_sem/
+To segment a single microscopy image, specify the path to the image to segment in the **-i** argument. For instance, to segment the SEM image **'77.png'** of the test dataset that has a pixel size of 0.07 micrometers, use the following command::
 
+    axondeepseg -t SEM -i test_segmentation/test_sem_image/image1_sem/77.png -s 0.07
 
-* To segment multiple images acquired with different resolutions,
-please use the folder structure explained in `Data <https://neuropoly.github.io/axondeepseg/documentation.html#data>`_,
-i.e., put all image with the same resolution in the same folder.
-* Then, segment each folder one after the other using the argument **-s** or segment all folders in one command
-by specifying multiple paths to segment and using a different pixel_size_in_micrometer.txt for each folder, like this::
+The script will use the explicitely specified size argument (here, 0.07) for the segmentation. If no pixel size is provided in the arguments, it will automatically read the image resolution encoded in the file **'pixel_size_in_micrometer.txt'** if that file exists in the folder containing the image to segment.
+The segmented acquisition will be saved in the same folder as the acquisition image, with the suffix **'_seg-axonmyelin.png'**, in *png* format, along with the binary axon and myelin segmentation masks (with the suffixes **'_seg-axon.png'** and **'_seg-myelin.png'**). In our example, the following output files will be generated: **'77_seg-axonmyelin.png'**, **'77_seg-axon.png'** and **'77_seg-myelin.png'**.
 
-    axondeepseg -t SEM -i test_sem_image/image1_sem test_sem_image/image2_sem/
+To segment the same image by using the **'pixel_size_in_micrometer.txt'** file in the folder (i.e. not specifying the pixel size as argument in the command), use the following command::
 
+    axondeepseg -t SEM -i test_segmentation/test_sem_image/image1_sem/77.png
 
-Here, we segment all images located in image1_sem and image2_sem that don't have the "segmented" suffix.
+Segment multiple images of the same resolution
```

Comment:
I would simply call it: "Segment multiple images" because this section also describes ways to segment images without the same resolution using the txt file